vpr/src/base/ShowSetup.cpp

```diff
@@ -61,21 +63,69 @@ void ShowSetup(const t_vpr_setup& vpr_setup) {
     }
 }
 
-void printClusteredNetlistStats() {
-    auto& device_ctx = g_vpr_ctx.device();
-    auto& cluster_ctx = g_vpr_ctx.clustering();
+void ClusteredNetlistStats::writeHuman(std::ostream& output) const {
+    output << "Netlist num_nets: " << num_nets << "\n";
```

Comment:
It would be good to write a first line in this file saying "Cluster level netlist and block usage statistics" so it's clear that num_nets etc. is from the cluster-level netlist.

Reply:
Added header line